adapter/CCXHelpers.c

```diff
@@ -80,7 +83,10 @@ void getNodeCoordinates( ITG * nodes, ITG numNodes, int dim, double * co, double
 	{
 		int nodeIdx = nodes[i] - 1;
 		//The displacements are added to the coordinates such that in case of a simulation restart the displaced coordinates are used for initializing the coupling interface instead of the initial coordinates
-		for( j = 0 ; j < dim ; j++ ) coordinates[i * dim + j] = co[nodeIdx * 3 + j] + v[nodeIdx * mt + j + 1];
+		for( j = 0 ; j < dim ; j++ )
+    {
+      coordinates[i * dim + j] = co[nodeIdx * 3 + j] + v[nodeIdx * mt + j + 1];
+    }
```

Comment:
Formatting looks wrong here.

Reply:
we could run precice's clang-format conf file on everything inside adapter/.